zproject/urls.py

```diff
@@ -725,6 +725,9 @@
     # API key - as we consider access to the API key sensitive
     # and just having a logged-in session should be insufficient.
     rest_path("users/me/api_key/regenerate", POST=regenerate_api_key),
+    #  This view accepts a JWT containing an email and returns an API key
+    #  and the details for a single user.
+    rest_path("api/v1/jwt/fetch_api_key", POST=(jwt_fetch_api_key, {"intentionally_undocumented"})),
```

Comment:
the path should be jwt/fetch_api_key, this creates a new endpoint api/v1/api/v1/jwt/fetch_api_key

Likely why the testsuite is failiing

```suggestion
    rest_path("jwt/fetch_api_key", POST=(jwt_fetch_api_key, {"intentionally_undocumented"})),
```

Reply:
Thanks for all the help, I've made the changes. Much appreciated.